### DISCUSSION TITLE
突然想起来有这么个网站）

### DISCUSSION THREAD

**Discussion by AbelScott1145** [share121/inter-knot#5358] [灌水]
退坑这游戏（zzz）有一段时间了，现在这游戏近况怎么样？剧情进度到哪里了？退坑了都没咋关注了
![Image_1727096060743](https://github.com/user-attachments/assets/7dd788c1-0eaf-4232-be28-30ad53413a15)

**Comment by baizuozhige** [comment]
长草中，前期由于逆天分配现在缺资源缺麻了

**Reply by 210flight** [reply]
确实是，体力都用来刷养成材料和丁尼了

**Comment by Firefly704** [comment]
明天法厄同就去外环冒险了，fairy主动帮忙拿奖励，加了后背隐藏能源，优化了不少

**Comment by owofile** [comment]
neuro可爱捏
